script/CommonDeployer.s.sol

```diff
@@ -37,18 +37,28 @@ struct CommonReport {
 }
 
 contract CommonActionBatcher {
-    // @dev lock to ensure we forbid calling engage/revoke methods after deployment
-    bool isLock;
+    error NotDeployer();
 
-    modifier unlocked() {
-        if (!isLock) _;
+    address deployer;
+
+    constructor() {
+        deployer = msg.sender;
+    }
```

Comment:
Forced to do this instead of passing a parameter due: https://github.com/ethereum/solidity/issues/4199